README.md

```diff
@@ -1,21 +1,22 @@
 # Sailfish QML
 Workplace repo for creating diff patches
 
-## How to manual?
+## How-to (fully manual)?
 - Fork repo or request addition to organization
-- Create branch named your patch name:
-`git checkout -b my-patch-name`
+- Create branch with your Patch name: `git checkout -b my-patch-name`
 - Apply changes
-- Create diff:
-`git diff master  -- . ':!.github' > unified_diff.patch`
-- Publish it to Patchmanager web catalog!
+- Create diff: `git diff master  -- . ':!.github' > unified_diff.patch`
+- Publish it on Patchmanager's Web Catalog!
 
-## How to automatic?
+## How-to (semi-automatic)?
 - Fork repo or request addition to organization
-- Create branch named your patch name:
-`git checkout -b my-patch-name`
+- Create branch with your Patch name: `git checkout -b my-patch-name`<br />
+  Alternatively this can be done by GitHub's web-frontend.
 - Apply changes
-- You can copy additional files according to [pm2 guidelines](https://coderus.openrepos.net/pm2/usage/) to the 'patch' folder
+- You can copy additional files according to the [Patchmanager guidelines](https://coderus.openrepos.net/pm2/usage/) to the `patch` folder
 - Create pull request
-- Github CI will produce zip file for you
-- Publish it to Patchmanager web catalog!
\ No newline at end of file
+- A CI workflow of this GitHub repo will [produce a ZIP file for you](https://github.com/sailfishos-patches/sailfish-qml/actions)
+- Publish it on Patchmanager's Web Catalog!
+
+#### Side note
+The content of this repo was created by executing `find /usr -name '*.qml' -o -name '*.js'` as root on a freshly installed SailfishOS.
```

Comment:
Content generated on Sailfish OS with every default application and all available qml plugin and declarative plugin installed

Reply:
Freshly installed sfos is not enough to retrieve all available qml files

Reply:
> Content generated on Sailfish OS with every default application and all available qml plugin and declarative plugin installed

O.K., so I have to improve this sentence.  A few checks & balances questions to determine, if I correctly understand what you mean:
- *every default application*
  I assume this addresses all applications which can be installed during the initial setup of SailfishOS ("first boot").  But does that include the proprietary components as `Weather` (i.e. Foreca Weather), `T9` ("words prediction"), `EAS` (Exchange Active Sync) and AlienDalvik (aka AAS)?
- *all available qml plugin […] installed*
  Sorry, I have no idea what this means.  When and where / how can these plug-ins be installed?  Which QML plug-ins fall into this category (maybe to name a few examples might help me to comprehend)?
- *all available […] declarative plugin installed*
  Sorry, again I have no idea what this means; I even do not know what a "declarative plugin" technically is.  Hence the questions from the prior point are also applicable here:
  - When and where / how can these plug-ins be installed?
  - Which "declarative" plug-ins fall into this category (maybe to name a few examples might help me to comprehend)?

Hopefully I manage to phrase something concise by the help of your reply.

Reply:
Enough to say like: installed every package available from system repositories providing qml and js files

Reply:
zypper se --provides qml
zypper se --provides js
zypper se --provides qmldir